packages/teleport-uidl-validator/src/decoders/utils.ts

```diff
@@ -345,7 +345,12 @@ export const elementStateDecoder: Decoder<UIDLElementStyleStates> = union(
   constant('hover'),
```

Comment:
```suggestion
export const elementStateDecoder: Decoder<UIDLElementStyleStates> = oneOf(
  constant('hover'),
```

and import `oneOf` instead of union. from `@mojotech/json-type-validation`